Source/Core/Core/CMakeLists.txt

```diff
@@ -798,11 +797,6 @@ if(MSVC)
   target_link_libraries(core PRIVATE use_pch)
 endif()
 
-if(USE_RETRO_ACHIEVEMENTS)
-  target_link_libraries(core PUBLIC rcheevos)
-  target_compile_definitions(core PUBLIC -DUSE_RETRO_ACHIEVEMENTS)
-  target_compile_definitions(core PUBLIC -DRC_CLIENT_SUPPORTS_HASH)
-  if(RC_CLIENT_SUPPORTS_RAINTEGRATION)
-    target_compile_definitions(core PUBLIC -DRC_CLIENT_SUPPORTS_RAINTEGRATION)
-  endif()
+if(NOT MSVC)
+  target_compile_options(core PRIVATE -Wno-invalid-offsetof)
 endif()
```

Comment:
Ditto regarding warning silencing. I have seen this warning silenced directly in several files (e.g. [Source/Core/Core/PowerPC.h](https://github.com/dolphin-emu/dolphin/blob/327d3fbd8f4e528aa67f7aba112e95cf98cb62c2/Source/Core/Core/PowerPC/PowerPC.h#L235-L238))